### DISCUSSION TITLE
[Feature] Prioritise photos (instead of videos) during background sync

### DISCUSSION THREAD

**Discussion by fredrikburmester** [immich-app/immich#3120] [Feature Request]
### The feature

**The app should prioritise backing up images instead of videos during the background sync.** 

The reasoning behind this is that videos can get quite large, and the background sync can only run between 10-30s at a time. This means that if a large video fails to upload in that time, nothing will get backed up. 

If photos were prioritised, they would at least get backed up, and the user could open the app to back up the large video asset instead.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by etnoy** [comment]
I agree with this suggestion; it's also what gphotos does

**Comment by bo0tzz** [comment]
Related:  #3341

**Comment by philffm** [comment]
100% agree. I was traveling for a month in China, mostly relying on mobile data. In those scenarios, it’s critical to back up photos first—they account for a fraction of the total size, so they're far less likely to cause transfer timeouts. Now after traveling for another month in other countries I am still 1000 media files / 30 days behind because some large videos (I wouldn't bother getting backed up later) keep on causing this traffic jam :D.

When your home server is 8,000 km away, latency and speed create a huge bottleneck (even on local WiFi), so getting the smaller photo assets backed up first makes a massive difference.
